shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -24,11 +24,19 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol FlutterPlugin <NSObject>
 @required
 /**
- Registers this plugin.
+ Registers this plugin with a plugin registry.
+
+ The plugin registry itself is represented during registration by the provided
```

Comment:
I had trouble parsing this sentence. From reading this, I'm not sure if the registry is the linkage between APIs and handlers held inside the registrar or if it is the registrar.